### ISSUE TITLE
Drupal subsite status report not showing correct profile

### ISSUE THREAD

**Issue by arun9989** [acquia/blt#2930]
My system information:
* Operating system type: Mac
* Operating system version: Sierra
* BLT version: BLT 9.1.0-alpha1

When I run this command:
```
blt setup --site=mysite
```

I get the following output:
```
In the sub site status report it showing installed profile as lightning.

```

I have installed a Drupal site with lightning profile and exported all the configuration in to /config/default folder using drush config-export command.

After that I have created a subsite(using `blt recipes:multisite:init `) and the subsite is installed with a custom profile which depends on lightning profile.

But after installation in subsite status report it is showing that Installed profile is 'lightning' instead of my custom profile.

Am I missing anything? Is there any additional parameters to be added in custom profile?

**Comment by ba66e77** [comment]
So "mysite" is the name of your default site? What's the name of your subsite and what command are you running to get the "subsite status report"?

**Comment by arun9989** [comment]
"mysite" is my subsite. I didn't change the name for default site.

**Comment by ba66e77** [comment]
First place I would check then is to make sure the "mysite" instance is connecting to the correct database. The multisite init process currently does not update the db credentials for a new site, so those have to be done manually (See the "Running...currently does not" section in http://blt.readthedocs.io/en/latest/multisite/#multisite).

**Comment by arun9989** [comment]
Yes, you are right. BLT does not update `docroot/sites/{newsite}/settings/local.settings.php`. But I updated this file manually. But this configuration is not picking up for my subsite. Subsite is loading db settings from `docroot/sites/default/settings/local.settings.php`. After update the db settings in default/settings/local.settings.php only subsite is loading with correct db. But still in status report it showing installed profile as lightning.

**Comment by ba66e77** [comment]
Did you also set up the sites.php file? That's another one that's in the "doesn't do" category but is needed for multisites.

If you're connecting to the correct database and the status report still shows Lightning as the profile, then I'd feel pretty confident that's what was setup on the site. How did you confirm the subsite was installed with the custom profile you expected?

**Comment by arun9989** [comment]
sites.php exist in docroot/sites/sites.php. I didn't made any change in this file. How I confirmed is my profile split is getting installed during setup also `blt setup` showing : 
`[Acquia\Blt\Robo\Tasks\DrushTask] Running /var/www/myproject/vendor/bin/drush @mysite.local site-install customprofile install_configure_form.update_status_module='array(FALSE,FALSE)' install_configure_form.enable_update_status_module=NULL --sites-subdir=mysite --site-name=MySite --site-mail=no-reply@acquia.com --account-name=ukTssgQkm8 --account-mail=no-reply@acquia.com --locale=en --uri=mysite -v --ansi in /var/www/myproject/docroot`

**Comment by lcatlett** [comment]
this was fixed in #3231
